maxprocs/maxprocs.go

```diff
@@ -71,6 +72,13 @@ func Min(n int) Option {
 	})
 }
 
+// RoundUpQuota controls whether the CPU quota should be rounded up instead of down.
+func RoundUpQuota(v bool) Option {
```

Comment:
Not a maintainer, but thinking out loud:
Should this maybe be an enum "Rounding"?

```
type Rounding int

const (
  RoundDown Rounding = iota
  RoundUp
)

func RoundQuota(Rounding) Option
```

Reply:
how about ceil and floor？

Reply:
Thanks @abhinav! Sounds good. I'm taking some time tomorrow to push that change and fix the codecov check

Reply:
Added. I had to add it in runtime to avoid having circular dependencies with maxprocs